### PR TITLE
LPS-164922 Sort the classes inside detectClickOutside function

### PR DESCRIPTION
**Pull request by rebsilva**
Related Issue: https://issues.liferay.com/browse/LPS-164922

This is a follow-up PR of this one https://github.com/liferay-forms/liferay-portal/pull/1991 since Brian asked to [sort the classes](https://github.com/brianchandotcom/liferay-portal/pull/125110#issuecomment-1287561999) inside the function. The original PR is already merged, so I'm just sending a commit sorting the classes, thanks!

### CODE REVIEW

**Comment by rebsilva** [comment]
ci:test:sf

File: modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/DataEngineLayoutBuilderHandler.es.js [hunk 1]
```diff
@@ -124,11 +124,11 @@ export default function DataEngineLayoutBuilderHandler({namespace}) {
 		if (
 			isElementInnerSelector(
 				target,
+				'.cke_dialog',
 				'.ddm-form-builder-wrapper',
 				'.multi-panel-sidebar',
 				'.lfr-icon-menu-open',
-				'.input-localized-content',
-				'.cke_dialog'
+				'.input-localized-content'
```
Comment (alinedoleron) [inline modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/DataEngineLayoutBuilderHandler.es.js:131]: Hi @rebsilva, just a question. Do you think this parameters should also be sorted?

Comment (rebsilva) [reply modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/DataEngineLayoutBuilderHandler.es.js:131]: Sure Aline! I forgot to sort them, thanks for telling me!

**Comment by rebsilva** [comment]
ci:stop

**Comment by rebsilva** [comment]
ci:test:relevant

**Review by alinedoleron — APPROVED** [review]
LGTM!

**Comment by alinedoleron** [comment]
ci:forward

**Pull request closed without merging**